src/Options.js

```diff
@@ -5,6 +5,25 @@
  * `.reset` functions.
  */
 
+const Style = require("./Style");
+
+const sizeStyleMap = [ // [textsize, scriptsize, scriptscriptsize]
```

Comment:
My understanding of KaTeX fonts is that we only have 10pt size fonts and we scale them appropriately.  What would prevent us from scaling that font to 6pt when necessary?